massgov/mayflower-patternlab
# DP 8594 overrides for organization row reorder for Elected Officials

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

The does not change display or functionality.  It only provides twig blocks to support overrides when implementing the reorder of component rows for the Elected Officials.

## Related Issue / Ticket

- [DP-8594](https://jira.state.ma.us/browse/DP-8594)

## Steps to Test


1. Checkout the branch locally and build. 
1. Confirm that the following hasn't changed:
    1. Event Listing Organism (by author): [local](http://localhost:3000/?p=organisms-event-listing) [production](https://mayflower.digital.mass.gov/?p=organisms-event-listing)
    1. Press Listing Organism (by author): [local](http://localhost:3000/?p=organisms-press-listing) [production](https://mayflower.digital.mass.gov/?p=organisms-press-listing)
    1. Section Three Up Organism (by author): [local](http://localhost:3000/?p=organisms-sections-three-up) [production](https://mayflower.digital.mass.gov/?p=organisms-sections-three-up)
    1. Section Links Molecule: [local](http://localhost:3000/?p=molecules-section-links) [production](https://mayflower.digital.mass.gov/?p=molecules-section-links)

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
